index.html

```diff
@@ -7,5 +7,17 @@
 	</head>
 	<body>
 		<h1>Hello, World</h1>
+		<hr />
+		<section id="About">
+			<h1>About</h1>
+			<p>
+				Lorem ipsum dolor sit amet consectetur, adipisicing elit. Non

```

Comment:
You should replace the placeholder text here

Reply:
Oh good catch! Resolved in the following commit!